cockroachdb/templates/job.init.yaml

```diff
@@ -1,4 +1,6 @@
-{{- if and (eq (len .Values.conf.join) 0) (not (index .Values.conf `single-node`)) }}
+{{ $isClusterInitEnabled := and (eq (len .Values.conf.join) 0) (not (index .Values.conf `single-node`)) }}
+{{ $isClusterProvisioningEnabled := .Values.init.provisioning.enabled }}
+{{- if or $isClusterInitEnabled $isClusterProvisioningEnabled }}
```

Comment:
Is it ok/possible for both of these values to be true? Do you ever want to run the provisioning if `$isClusterInitEnabled` is false?

Reply:
Yes, for single node with preconfigured databases

Reply:
I think i can clarify usage and rename to $isDatabaseProvisioningEnabled